### PR TITLE
8ball: add the spooky magic 8 ball

### PR DESCRIPTION
**Pull request by Exirel**
Tin. This PR is open for feedback!

### CODE REVIEW

File: sopel_8ball/choices.py [hunk 1]
```diff
@@ -118,3 +118,37 @@ def choices(self):
...
+        return (
+            # affirmative answers
+            "The ghost over your shoulder said yes.",
+            "The accursed screams in agreement.",
+            "The elders smile from the deep.",
+            "I've seen this in the dreams of a Great Old One.",
+            "For once eerie voices whisper in harmony.",
+
+            # non-committal answers
+            "The abyss remain silent.",
```
Comment (dgw) [inline sopel_8ball/choices.py:135]: Unless "the abyss" is a collective of beings? But I doubt that.
```suggestion
            "The abyss remains silent.",
```

File: sopel_8ball/choices.py [hunk 2]
```diff
@@ -118,3 +118,37 @@ def choices(self):
...
+            "I hear the whispers of a thousand voices "
+            "but none have your answer.",
+            "The unfathomable truth cannot be revealed.",
+            "In the darkness this won't matter anymore.",
+            "From the depth comes no answer.",
+
+            "The spirits are bored of this tune.",
+            "The answer lies behond the stars, ineffable.",
```
Comment (dgw) [inline sopel_8ball/choices.py:143]:
```suggestion
            "The answer lies beyond the stars, ineffable.",
```

File: sopel_8ball/choices.py [hunk 3]
```diff
@@ -118,3 +118,37 @@ def choices(self):
...
+            "Answers are lost in the midst of black seas of infinity.",
+            "You shall go mad from the revelation of the truth.",
+            "I hear your call yet their shriek keeps me silent.",
```
Comment (dgw) [inline sopel_8ball/choices.py:146]:
```suggestion
            "I hear your call, yet their shrieking keeps me silent.",
```

This is stylistic, not necessarily grammatical.

File: sopel_8ball/choices.py [hunk 4]
```diff
@@ -118,3 +118,37 @@ def choices(self):
...
+
+            # negative answers
+            "This would angry the old gods.",
```
Comment (dgw) [inline sopel_8ball/choices.py:149]:
```suggestion
            "This would anger the old gods.",
```

File: sopel_8ball/choices.py [hunk 5]
```diff
@@ -118,3 +118,37 @@ def choices(self):
...
+            "This will bring sorrow and tears.",
+            "The elders scowl from the deep.",
+            "Hidden and fathomless minds pulsated in rage.",
```
Comment (dgw) [inline sopel_8ball/choices.py:152]:
```suggestion
            "Hidden and fathomless minds pulsate in rage.",
```

Most of these aren't in past tense, and I don't see a reason for this one to be. 😀

File: sopel_8ball/choices.py [hunk 6]
```diff
@@ -118,3 +118,37 @@ def choices(self):
...
+            "The underworld grumbles with dissension.",
```
Comment (dgw) [inline sopel_8ball/choices.py:153]: Dissension is more like not agreeing on the answer than saying no.

```suggestion
            "The underworld grumbles with repudiation.",
```

```suggestion
            "The underworld grumbles with disaffirmation.",
```

File: tests/test_choices.py [hunk 7]
```diff
@@ -27,3 +27,11 @@ def test_snarky():
 
     assert len(choices) == 20
     assert provider.query(None, None) in choices
+
+
+def test_classic():
```
Comment (dgw) [inline tests/test_choices.py:32]: m8, you're redefining the same function name :)

```suggestion
def test_spooky():
```

File: tests/test_managers.py [hunk 8]
```diff
@@ -68,3 +69,25 @@ def test_manager_setup_snarky(configfactory, botfactory):
...
+owner = testnick
+nick = TestBot
+enable = coretasks, 8ball
+
+[magic8ball]
+choices = spooky
+"""
+
+
+def test_manager_setup_snarky(configfactory, botfactory):
```
Comment (dgw) [inline tests/test_managers.py:85]: Ditto.

```suggestion
def test_manager_setup_spooky(configfactory, botfactory):
```

**Review by dgw — COMMENTED** [review]
Not Just Grammar®: a dgw review

File: sopel_8ball/choices.py (hunk 1 above)
Comment (Exirel) [reply sopel_8ball/choices.py:135]: After some digging, it's clearly a very fr to en translation problem.

**Pull request merged**